projects/experimental/directives/progress-segmented/progress-segmented.module.ts

```diff
@@ -0,0 +1,14 @@
+import {NgModule} from '@angular/core';
+
+import {TuiProgressSegmentedComponent} from './progress-segmented.component';
+import {TuiProgressSegmentedDirective} from './progress-segmented.directive';
+
+/**
+ * TODO: Move to TuiProgressModule (from `@taiga-ui/kit`) in 4.0
+ * TODO: delete the previous version of the same component
+ */
+@NgModule({
+    declarations: [TuiProgressSegmentedComponent, TuiProgressSegmentedDirective],
+    exports: [TuiProgressSegmentedComponent, TuiProgressSegmentedDirective],
```

Comment:
I don't think you need to export the component

Reply:
✅